Source/plugins/JSONRPC.h

```diff
@@ -437,10 +437,10 @@ namespace PluginHost {
             else if (realMethod == _T("exists")) {
                 result = Core::ERROR_NONE;
                 if (handler->Exists(parameters) == Core::ERROR_NONE) {
-                    response = _T("1");
```

Comment:
This will deviate from what they are expecting in R2 I think, So I assume this is now a breaking interface with R2 (and R4.2)? Right, if so we need to keep it as it was and indeed for R5 we can report a breaking interface change here, if we would like to break it. (Maybe something for the CCB)

Reply:
This is added to make sure it sync with R4.2 version : https://github.com/rdkcentral/Thunder/blob/R4_2/Source/plugins/JSONRPC.h#L352-L356